rfcs/042-editable-programmatic-descriptions.md

```diff
@@ -0,0 +1,100 @@
+- Feature Name: editable_programmatic_descriptions
+- Start Date: 2021-07-27
+- RFC PR: [amundsen-io/rfcs#42](https://github.com/amundsen-io/rfcs/pull/42)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000)
+
+# Editable Programmatic Descriptions
+
+## Summary
+
+Programmatic descriptions are a useful way for Amundsen users to store and display business specific metadata about their tables in Amundsen. This feature allows for some customization of Amundsen without customizing the underlying software. Currently, this feature focuses on the machine generated metadata use case. This type of data can be extracted from the source system of truth and ingested into Amundsen via databuilder.
+
+The same need exists for business specific human generated metadata. This type of data does not exist in a machine generated form that can be easily extracted from a source system. Instead, users need to add/edit this metadata directly in the Amundsen UI. 
```

Comment:
for the case that programmatic description are machine generated vs manual edit, how do you enable/disable the edit for each?

Reply:
ok, saw it in the later section that you config through FE config.

Reply:
I wonder if it would make sense to make these "programmatic descriptions", as they wouldn't be "programmatic" (except the seeded ones).

I think these could be called Human Generated Descriptions or a similar name.